### PR TITLE
Add tools to help debug image GC

### PR DESCRIPTION
**Pull request by Mm2PL**
# Description

Add:
- `/force-image-gc` which calls the image gc code normally, just as if its timer were to fire,
- `/force-image-unload` which clears all the images tracked by the gc,
- `image bytes`, `image bytes (ever {un,}loaded)` counters which try to visualize memory usage,
- `images tracked by gc` counter which shows how many total images GC has (including empty ones),
- `last image gc eligible` counter which shows how many images were eligible for GC,
- `last image gc expired` counter which shows how many images were evicted from memory.

### CODE REVIEW

File: src/controllers/commands/CommandController.cpp [hunk 1]
```diff
@@ -3207,6 +3209,28 @@ void CommandController::initialize(Settings &, Paths &paths)
 
         return "";
     });
+
+    this->registerCommand(
+        "/force-image-gc",
+        [](const QStringList & /*words*/, auto /*channel*/) -> QString {
+            postToThread([]() {
```
Comment (Nerixyz) [inline src/controllers/commands/CommandController.cpp:3216]: It is extremely unlikely that this runs outside the gui-thread (e.g. in `execCommand` there's a call to `getApp`), so this should use `runInGuiThread`.

File: src/controllers/commands/CommandController.cpp [hunk 2]
```diff
@@ -3207,6 +3209,28 @@ void CommandController::initialize(Settings &, Paths &paths)
...
+                auto &iep = ImageExpirationPool::instance();
+                iep.freeOld();
+            });
+            return "";
+        });
+
+    this->registerCommand(
+        "/force-image-unload",
+        [](const QStringList & /*words*/, auto /*channel*/) -> QString {
+            postToThread([]() {
```
Comment (Nerixyz) [inline src/controllers/commands/CommandController.cpp:3227]: Same here.

File: src/util/DebugCount.hpp [hunk 3]
```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "common/UniqueAccess.hpp"
+#include "util/Helpers.hpp"
```
Comment (pajlada) [inline src/util/DebugCount.hpp:4]: shouldn't need this include

**Review by pajlada — COMMENTED** [review]
looks sane to me

**Review by dnsge — APPROVED** [review]

**Review by pajlada — APPROVED** [review]

**Pull request merged**